boards/sparc/bm3803/xx3803/scripts/Make.defs

```diff
@@ -55,7 +55,7 @@ ifeq ($(CONFIG_DEBUG_SYMBOLS),y)
 endif
 
 ifneq ($(CONFIG_DEBUG_NOOPT),y)
-  ARCHOPTIMIZATION += $(MAXOPTIMIZATION) -fno-strict-aliasing -fomit-frame-pointer
+  ARCHOPTIMIZATION += $(MAXOPTIMIZATION) -fno-strict-aliasing
```

Comment:
Do we need to add `ifeq ($(CONFIG_FRAME_POINTER),y)` case here?

Reply:
Don't need since I add to Toolchain.defs here:
https://github.com/apache/incubator-nuttx/pull/5282/files#diff-956cdffe92103d6a46573f33066d5f0c0b4961f507a1defb5682c26cbeb544bbR82-R86